selfdrive/car/toyota/values.py

```diff
@@ -925,7 +938,6 @@ class CAR:
       b'\x018966353Q4000\x00\x00\x00\x00',
     ],
     (Ecu.engine, 0x7e0, None): [
-      b'\x0232480000\x00\x00\x00\x00\x00\x00\x00\x00A4701000\x00\x00\x00\x00\x00\x00\x00\x00',
```

Comment:
why was this removed?

Reply:
This was the exact engine fw on @bisra2's ISH. This firmware was added in https://github.com/commaai/openpilot/pull/20850 as part of a batch. It seemed extremely unlikely that the gas-only model has the same engine firmware as the hybrid and I think this was an error.

Reply:
FWIW, I think we might see a similar situation with the XX40 Avalons and XX50 Avalons. I want to send a PR in to separate those outs later but firmwares from users who have hacked their installations to get XX50 Avalons working by changing the DBCs used are still identifying as the presumably XX40 Avalon. 

I think something similar happened here for the ISH since @bisra2 was running a hacked version that was still identifying as an IS before I made this proper separated-out branch.